common/djangoapps/student/management/commands/transfer_students.py

```diff
@@ -52,7 +52,6 @@ class Command(TrackedCommand):
                     help="If True, try to transfer certificate items to the new course.")
     )
 
-    @transaction.commit_manually
```

Comment:
@macdiesel "if command fails on a user part way through the transaction" `atomic()` will 1. catch the exception, 2. rollback the transaction and 3. re-raise the exception. And yes, that means some of the students would have been transferred and the others not. But this is how it currently works too.

Actually, in Django 1.4 `commit_manually` is only used to disable automatic transaction management for views. Commands do not need it. So its not doing anything right now either. 
https://docs.djangoproject.com/en/1.4/topics/db/transactions/#django.db.transaction.commit_manually